src/commands/enroll-data/GetOverallEnroll.ts

```diff
@@ -42,33 +44,64 @@ export default class GetOverallEnroll extends BaseCommand {
                 break;
         }
 
-        const arr = coll.get(term);
-        if (!arr) {
-            await ctx.interaction.reply({
-                content: `The term, **\`${term}\`** (Display \`${display}\`), could not be found. Try again.`,
-                ephemeral: true,
-            });
+        let allPlots;
```

Comment:
is there a reason why you're defining this variable here? why not in the scope where it's being assigned?